### PR TITLE
docs: update the readme

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**

### CODE REVIEW

**Review by tiagolobocastro — APPROVED** [review]

**Review by orville-wright — APPROVED** [review]

**Review by avishnu — APPROVED** [review]

**Comment by edrob999** [comment]
1. The opening paragraph needs to have the title "Project Purpose".  
2. Do we need the mule picture at the top of the readme? If you guys think this is important, then include it. Otherwise, it uses up half-a-page, remove it

File: README.md [hunk 1]
```diff
@@ -1,52 +1,61 @@
...
-| [<img src="https://github.com/openebs/community/blob/develop/images/local-pv_rawfile_183x163_t.png" alt="Local PV Rawfile" align="center" width="83px"/>](https://github.com/openebs/rawfile-localpv) |  [Local PV Rawfile](https://github.com/openebs/rawfile-localpv)    | for integration with Loop mounted Raw device-file | Beta, deployable in testing, undergoing evaluation & integration <BR>```release: v0.70```   | v4.1.0   |
-<BR>
-
-STANDARD is optimized for NVMe and SSD Flash storage media, and integrates ultra modern cutting-edge high performance storage technologies at its core...</summary>
+STANDARD is optimized for NVMe and SSD Flash storage media and integrates ultra-modern cutting-edge high-performance storage technologies at its core...</summary>
```
Comment (avishnu) [inline README.md:50]: I think we should do away with "STANDARD", there's just OpenEBS now.

File: README.md [hunk 2]
```diff
@@ -1,52 +1,61 @@
...
 
 <BR>
 
 ## Monthly Community Meetings
-TThe community meetings are usually held monthly on the last thursday of the month at 14:00 UTC.<BR>
-The meeting is currently held on Zoom: https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1<BR>
-Upcoming meeting: 29 Aug 2024 at 14:00 UTC
+OpenEBS holds a monthly community meeting via Zoom on the last Thursday of the month, at 7 am Pacific time.
+<br>
+The next meeting is on: `Thursday 26 September, at 14:00 UTC`
```
Comment (niladrih) [inline README.md:13]: This is difficult to maintain. Maybe the meeting link is enough?
The mention of Pacific time and UTC is rather confusing. Can we stick with either of the two?

Comment (niladrih) [reply README.md:13]: Also, with Pacific time, the exact time is sometimes difficult to tell when daylight savings is on. My recommendation would be UTC.

Comment (Abhinandan-Purkait) [reply README.md:13]: Sure. I would change it to UTC.

File: README.md (hunk 1 above)
Comment (Abhinandan-Purkait) [reply README.md:50]: Done.

**Review by niladrih — APPROVED** [review]

**Pull request merged**